### PR TITLE
Added Logo to README

### PR DESCRIPTION
**Pull request by gautamjajoo**
Updated README.md by adding the logo!!
@AvaisP

### CODE REVIEW

**Comment by AvaisP** [comment]
@gautamjajoo reduce the size of the logo

File: README.md [hunk 1]
```diff
@@ -1,5 +1,7 @@
 # Origami
 
+<p align="center"><img width="65%" src="origami.png" /></p>
```
Comment (AvaisP) [inline README.md]: reduce the height as well

**Comment by gautamjajoo** [comment]
@AvaisP Reduced the size of Logo..

**Comment by AvaisP** [comment]
Good job!

**Pull request merged**